### PR TITLE
feature(e2e): When no select test testRun failed

### PR DESCRIPTION
**Pull request by jesusvallez**
If run test cafe without test to execute, cli returns error like: 

```
/Users/jesus/Documents/wjs/masmovil/frontend3/node_modules/cucumber-testcafe/src/lib/test-controller-holder.ts:19
      testControllerHolder.lastTestController.testRun
                                              ^
TypeError: Cannot read property 'testRun' of null
    at Object.testDone (/Users/jesus/Documents/wjs/masmovil/frontend3/node_modules/cucumber-testcafe/src/lib/test-controller-holder.ts:19:47)
    at Timeout.checkLastResponse [as _onTimeout] (/Users/jesus/Documents/wjs/masmovil/frontend3/node_modules/cucumber-testcafe/src/lib/hooks.ts:92:30)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
error Command failed with exit code 1.
```

### CODE REVIEW

File: src/lib/hooks.ts [hunk 1]
```diff
@@ -24,12 +23,10 @@ function createTestFile(featureName = '', scenarioName = '') {
   )
 }
 
-function runTest(iteration, browser) {
+function runTest(browser) {
   let runner
 
-  testCafe('localhost', 1338 + iteration, 1339 + iteration).then(function(
```
Comment (jesusvallez) [inline src/lib/hooks.ts:30]: We don't need iteration to generate random ports. If you don't define it, testcafe will set differents ports for you (pararell option)

File: src/lib/hooks.ts [hunk 2]
```diff
@@ -39,8 +36,7 @@ function runTest(iteration, browser) {
       .run({
         skipJsErrors: true,
         selectorTimeout: TIMEOUT - TIMEOUT / 5,
-        assertionTimeout: TIMEOUT / 2,
-        speed: 1
```
Comment (jesusvallez) [inline src/lib/hooks.ts:43]: speed 1 is default, we dont need to define it

File: src/lib/hooks.ts [hunk 3]
```diff
@@ -51,22 +47,20 @@ function runTest(iteration, browser) {
...
-  await this.waitForTestController.then(t => {
-    return t.maximizeWindow()
-  })
+  runTest(process.env.CUCUMBER_BROWSER || this.parameters.browser)
+
+  await this.waitForTestController.then(t => t.maximizeWindow())
 })
 
 After(function() {
-  unlinkSync(RUNNER_FILE)
```
Comment (jesusvallez) [inline src/lib/hooks.ts:69]: Check if exist before to remove RUNNER_FILE

File: src/lib/hooks.ts [hunk 4]
```diff
@@ -89,15 +83,17 @@ AfterAll(function() {
   }
 
   function checkLastResponse() {
-    if (testControllerHolder.testDone()) {
```
Comment (jesusvallez) [inline src/lib/hooks.ts:92]: If testControllerHolder dont exist, testcontroller return error

```
/Users/jesus/Documents/wjs/masmovil/frontend3/node_modules/cucumber-testcafe/src/lib/test-controller-holder.ts:19
      testControllerHolder.lastTestController.testRun
                                              ^
TypeError: Cannot read property 'testRun' of null
    at Object.testDone (/Users/jesus/Documents/wjs/masmovil/frontend3/node_modules/cucumber-testcafe/src/lib/test-controller-holder.ts:19:47)
    at Timeout.checkLastResponse [as _onTimeout] (/Users/jesus/Documents/wjs/masmovil/frontend3/node_modules/cucumber-testcafe/src/lib/hooks.ts:92:30)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
error Command failed with exit code 1.
```

File: package.json [hunk 5]
```diff
@@ -33,7 +33,6 @@
     "@types/cucumber": "^4.0.7",
     "base64-img": "^1.0.4",
     "cucumber": "^5.1.0",
-    "cucumber-junit-convert": "^1.1.0",
```
Comment (jesusvallez) [inline package.json:39]: junit unneeded dependencies

**Review by anthanh — APPROVED** [review]
LGTM

**Pull request merged**